src/main/java/org/jenkinsci/plugins/p4/client/Validate.java

```diff
@@ -25,13 +25,28 @@ public Validate(TaskListener listener) {
 	 *
 	 * @param fileSpecs List of Perforce file specs
 	 * @param ignore    Parameter list of messages to ignore
-	 * @throws Exception push up stack
 	 * @return true if no errors.
+	 * @throws Exception push up stack
 	 */
 	public boolean check(List<IFileSpec> fileSpecs, String... ignore) throws Exception {
 		return check(fileSpecs, true, ignore);
 	}
 
+	/**
+	 * Only return boolean; false is returned for an exception.
+	 *
+	 * @param fileSpecs List of Perforce file specs
+	 * @param ignore    Parameter list of messages to ignore
+	 * @return true if no errors or exceptions
+	 */
+	public boolean checkCatch(List<IFileSpec> fileSpecs, String... ignore) {
+		try {
+			return check(fileSpecs, true, ignore);
+		} catch (Exception e) {
```

Comment:
The command has already run at this point.  The Validate class and various methods are to check the messages from Perforce.  It's not great code, but the check method was raising the exception.  Really I should change the logic inside the check method such that it doesn't raise an exception.